src/components/measureEditor/MeasureEditor.tsx

```diff
@@ -105,8 +88,16 @@ const MeasureEditor = () => {
   };
 
   useEffect(() => {
-    debouncedChangeHandler(editorVal);
-  }, [editorVal, debouncedChangeHandler]);
+    if (!editorVal) {
+      updateElmAnnotations(measure.cql).catch((err) => {
+        console.error("An error occurred while translating CQL to ELM", err);
+        setElmTranslationError("Unable to translate CQL to ELM!");
+        setElmAnnotations([]);
+      });
+      setEditorVal(measure.cql);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [editorVal]);
```

Comment:
eslint is suggesting to use "updateElmAnnotations" as a dependency, but whenever a state is changed because of "updateElmAnnotations" useeffect gets called, which is not ideal and also may put the component in an infinite loop. 
One way to resolve this is to use "useCallback" hook on the called function ( updateElmAnnotations() ), doing which is not solving the issue. This is already implemented in a few other components (ex: inlineEditt) of our project and I guess finally got ignored.